script/lint

```diff
@@ -17,5 +17,5 @@ fi
 
 SOURCES="*.py octodns/*.py octodns/*/*.py tests/*.py"
 
-pep8 --ignore=E221,E241,E251 $SOURCES
+pycodestyle --ignore=E221,E241,E251,E722 $SOURCES
```

Comment:
👍, though ideally there wouldn't be bare `except`s. Glad this catches them now, I may go through and look at addressing the existing warnings and reenabling this at some point. I probably would have said something about them if I'd noticed in the original PRs as I prefer to avoid them.